docs/extending.md

```diff
@@ -289,14 +291,34 @@ from caluma.caluma_workflow.dynamic_groups import (
 
 class CustomDynamicGroups(BaseDynamicGroups):
     @register_dynamic_group("legal_dept")
-    def resolve_legal_dept(self, task, case, user, prev_work_item):
+    def resolve_legal_dept(self, task, case, user, prev_work_item, _context):
         # custom business logic which returns either a string or a list of strings
         return find_legal_dept(case)
 ```
 
 You can then use that defined dynamic group `"legal_dept"` in the `Task` properties by postfixing them with a transform: `"legal_dept"|groups`.
 The created `WorkItem` would then have an evaluated value like `["legal-dept-boston"]` or `["legal-dept-nyc"]` for that property.
 
+The `"_context"` can hold an arbitrary dict passed in the `"_context"` variable in a `SaveCase` or `CompleteWorkItem` mutation.
+Use it for passing additional info and implement logic on it, for example run-time dependent `"addressed_groups"` that should be added to newly generated `WorkItems`.
+
+
+We can extend our example above:
+
+```python
+class CustomDynamicGroups(BaseDynamicGroups):
+    @register_dynamic_group("legal_dept")
+    def resolve_legal_dept(self, task, case, user, prev_work_item, _context):
+        # _context might contain additional filters...
+        legal_depts = find_legal_dept(case, coast=_context.get("coast"))
+
+        # ... or plain groups to include
+        return legal_depts + _context.get("additional_groups", [])
+```
+
+Now if the `_context` looks like `{"coast": "west", "additional_groups": ["board", "administration"]}`, the resulting groups are `["legal-dept-san-francisco", "legal-dept-seattle", "board", "administration"]`.
```

Comment:
I like the example! However here's some leftover underscore

Reply:
Done